frontend/src/config/bhutan/layers.json

```diff
@@ -379,8 +379,193 @@
+      {
+        "value": 750,
+        "label": "750 - 999",
+        "color": "#31a354"
+      },
+      {
+        "value": 1000,
+        "label": ">= 1000",
+        "color": "#006d2c"
+      }
+    ],
+    "legend_text": "Source: Population and Housing Census of Bhutan, 2017"
+  },
+  "age_below15": {
+    "title": "Population under 15 years old",
+    "type": "admin_level_data",
+    "path": "data/bhutan/btn_adm3_level_data.json",
+    "data_field": "AgeBelow15",
+    "admin_level": 3,
+    "admin_code": "Adm3_Code",
+    "opacity": 0.8,
+    "legend": [
+      {
+        "value": 0,
+        "label": "1 - 249",
+        "color": "#edf8e9"
+      },
+      {
+        "value": 250,
+        "label": "250 - 499",
+        "color": "#bae4b3"
+      },
+      {
+        "value": 500,
+        "label": "500 - 749",
+        "color": "#74c476"
+      },
+      {
+        "value": 750,
+        "label": "750 - 999",
+        "color": "#31a354"
+      },
+      {
+        "value": 1000,
+        "label": ">= 1000",
+        "color": "#006d2c"
+      }
+    ],
+    "legend_text": "Source: Population and Housing Census of Bhutan, 2017"
+  },
+  "age15_64": {
+    "title": "Population 15-64 years old",
+    "type": "admin_level_data",
+    "path": "data/bhutan/btn_adm3_level_data.json",
+    "data_field": "Age15_64",
+    "admin_level": 3,
+    "admin_code": "Adm3_Code",
+    "opacity": 0.8,
+    "legend": [
+      {
+        "value": 0,
+        "label": "1 - 249",
+        "color": "#edf8e9"
+      },
+      {
+        "value": 250,
+        "label": "250 - 499",
+        "color": "#bae4b3"
+      },
+      {
+        "value": 500,
+        "label": "500 - 749",
+        "color": "#74c476"
+      },
+      {
+        "value": 750,
+        "label": "750 - 999",
+        "color": "#31a354"
+      },
+      {
+        "value": 1000,
+        "label": ">= 1000",
+        "color": "#006d2c"
+      }
+    ],
+    "legend_text": "Source: Population and Housing Census of Bhutan, 2017"
+  },
+  "age_over64": {
+    "title": "Population over 64 years old",
+    "type": "admin_level_data",
+    "path": "data/bhutan/btn_adm3_level_data.json",
+    "data_field": "AgeAbove65",
+    "admin_level": 3,
+    "admin_code": "Adm3_Code",
+    "opacity": 0.8,
+    "legend": [
+      {
+        "value": 0,
+        "label": "1 - 24",
+        "color": "#edf8e9"
+      },
+      {
+        "value": 25,
+        "label": "25 - 49",
+        "color": "#bae4b3"
+      },
+      {
+        "value": 50,
+        "label": "50 - 74",
+        "color": "#74c476"
+      },
+      {
+        "value": 75,
+        "label": "75 - 99",
+        "color": "#31a354"
+      },
+      {
+        "value": 100,
+        "label": ">= 100",
+        "color": "#006d2c"
+      }
+    ],
+    "legend_text": "Source: Population and Housing Census of Bhutan, 2017"
+  },
   "rice": {
-    "title": "Rice - 274g / head / day in Kg Unit",
+    "title": "Rice daily per capita consumption in grams",
```

Comment:
I'm not sure that this title reads right to me. Could maybe use a colon? `Rice: daily per capita consumption in grams`. I'll happily defer to you